ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.scss

```diff
@@ -0,0 +1,156 @@
+@use "../../../../styles/FON/material/dialog" as angMatDialog;
+@use "../../../../styles/FON/classes/buttons";
+@import "../../../../styles/FON/core";
+
+
+@include angMatDialog.mainSet(RegisterPatientsModalComponent);
+
+.dialogContainer {
+  height: 100%;
+  width: 100%;
+  display: grid;
+  grid-template-rows: 8.8% 79.2% 12%;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+
+  header {
+    grid-area: header;
+    display: flex;
+    align-items: center;
+    border-bottom: 0.5px inset rgba(148, 147, 147, 0.63);
+    padding: 6px 14.5px;
+
+    p {
+      color: #2D333E;
```

Comment:
Why would not use this color (and others in the file) from colors varilables ? (`_colors.scss`)

Reply:
I don't have yet refactored everything after I implemented global variables